sys/include/net/ng_netconf.h

```diff
@@ -119,8 +119,12 @@ typedef enum {
      * @note not all transceivers may support this interrupt
      */
     NETCONF_OPT_TX_END_IRQ,
-    NETCONF_OPT_AUTOCCA,            /**< en/disable to check automatically
-                                         before sending the channel is clear. */
+    NETCONF_OPT_CSMA,               /**< en/disable to check automatically
```

Comment:
I also added `NETCONF_OPT_CSMA_RETRIES` and to make clear that this is linked to `NETCONF_OPT_AUTOCCA` I figured that it should be renamed. Only the kw2xrf is using this option and this board also does CSMA. It's more precise and googling `AUTOCCA` only yields rubbish. 
But I see that renaming legacy options might not be a good idea per se.